packages/web/src/hooks/useSyntaxHighlightingExtension.ts

```diff
@@ -2,20 +2,68 @@
 
 import { EditorView } from "@codemirror/view";
 import { useExtensionWithDependency } from "./useExtensionWithDependency";
-import { javascript } from "@codemirror/lang-javascript";
-import { python } from "@codemirror/lang-python";
-import { markdown } from "@codemirror/lang-markdown";
+import { StreamLanguage } from "@codemirror/language";
+
+// CodeMirror 6 languages
+import { css } from "@codemirror/lang-css";
 import { cpp } from "@codemirror/lang-cpp";
 import { csharp } from "@replit/codemirror-lang-csharp";
-import { json } from "@codemirror/lang-json";
+import { go } from "@codemirror/lang-go";
+import { html } from "@codemirror/lang-html";
 import { java } from "@codemirror/lang-java";
+import { javascript } from "@codemirror/lang-javascript";
+import { json } from "@codemirror/lang-json";
+import { less } from "@codemirror/lang-less";
+import { liquid } from "@codemirror/lang-liquid";
+import { markdown } from "@codemirror/lang-markdown";
+import { php } from "@codemirror/lang-php";
+import { python } from "@codemirror/lang-python";
 import { rust } from "@codemirror/lang-rust";
-import { go } from "@codemirror/lang-go";
+import { sass } from "@codemirror/lang-sass";
 import { sql } from "@codemirror/lang-sql";
-import { php } from "@codemirror/lang-php";
-import { html } from "@codemirror/lang-html";
-import { css } from "@codemirror/lang-css";
+import { vue } from "@codemirror/lang-vue";
+import { xml } from "@codemirror/lang-xml"
+import { yaml } from "@codemirror/lang-yaml";
 
+// Legacy CodeMirror 5 modes
+import { clojure } from "@codemirror/legacy-modes/mode/clojure";
+import { cmake } from "@codemirror/legacy-modes/mode/cmake";
+import { cobol } from "@codemirror/legacy-modes/mode/cobol";
+import { coffeeScript } from "@codemirror/legacy-modes/mode/coffeescript";
+import { commonLisp } from "@codemirror/legacy-modes/mode/commonlisp";
+import { d } from "@codemirror/legacy-modes/mode/d";
+// import { dart } from "@codemirror/legacy-modes/mode/dart";
+// import { django } from "@codemirror/legacy-modes/mode/django";
+import { dockerFile } from "@codemirror/legacy-modes/mode/dockerfile";
+import { diff } from "@codemirror/legacy-modes/mode/diff";
+import { elm } from "@codemirror/legacy-modes/mode/elm";
+import { erlang } from "@codemirror/legacy-modes/mode/erlang";
+import { fortran } from "@codemirror/legacy-modes/mode/fortran";
+import { groovy } from "@codemirror/legacy-modes/mode/groovy";
+import { haskell } from "@codemirror/legacy-modes/mode/haskell";
+import { jinja2 } from "@codemirror/legacy-modes/mode/jinja2";
+import { julia } from "@codemirror/legacy-modes/mode/julia";
+import { liveScript } from "@codemirror/legacy-modes/mode/livescript";
+import { lua } from "@codemirror/legacy-modes/mode/lua";
+import { nginx } from "@codemirror/legacy-modes/mode/nginx";
+// import { octave } from "@codemirror/legacy-modes/mode/octave";
```

Comment:
These exist on the https://codemirror.net/5/mode/ page but are seemingly missing from the import.

Reply:
That's strange - looks like it's here? https://github.com/codemirror/legacy-modes/blob/main/mode/octave.js

Reply:
Fixed and added